src/firefly/js/ui/FileUploadProcessor.jsx

```diff
@@ -49,31 +49,39 @@ const tableWarning = 'Only loading the image(s), ignoring any selected table(s).
 
 export function resultSuccess(request) {
 
-    const {currentReport, currentDetailsModel, summaryModel, groupKey, acceptList, uniqueTypes}= request.additionalParams ?? {};
+    const {report, detailsModel, summaryModel, groupKey, acceptList, uniqueTypes,
+        acceptOneItem}= request.additionalParams ?? {};
     const summaryTblId = groupKey; //FileUploadAnalysis
     const fileCacheKey = getFileCacheKey(groupKey);
 
+    //highlighted row and entryType is used when acceptOneItem === true
+    const highlightedRowIdx = summaryModel?.highlightedRow;
+    const highlightedRow = summaryModel?.tableData?.data?.[highlightedRowIdx];
+    const entryType = highlightedRow?
+        highlightedRow[1]: summaryModel?.tableData?.data?.[0]?.[1];
+
     //determine if the file type or selections are valid to be loaded
-    const {valid, errorMsg, title} = determineValidity(acceptList, uniqueTypes, summaryModel, summaryTblId, currentReport);
+    const {valid, errorMsg, title} = determineValidity(acceptList, uniqueTypes, summaryModel,
+        summaryTblId, report, acceptOneItem, highlightedRowIdx, entryType);
```

Comment:
you are passing `determineValidity` the summary model already. Pull this out in the function and keep result sucess clean.